include/grpcpp/support/server_callback.h

```diff
@@ -477,7 +477,7 @@ class ServerBidiReactor : public internal::ServerReactor {
   }
 
   grpc::internal::Mutex stream_mu_;
-  // TODO(vjpai): Make stream_or_backlog_ into a std::variant or absl::variant
+  // TODO(vjpai): Make stream_or_backlog_ into a std::variant or std::variant
```

Comment:
Don't need to mention `std::variant` twice. :)

Reply:
Fixed, but I'm not sure what this TODO is referring to, `stream_or_backlog_` isn't a thing anymore. Given that this is a suggestion for a refactoring, I'm going to leave this as is for now, but it seems like either a thing we can do as part of a c++17 cleanup, or a TODO we can delete.